MonstieWash/Assets/Scripts/Compendium/CompendiumEntry.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.EventSystems;
+
+public class CompendiumEntry : MonoBehaviour
+{
+    public string MonsterName { get => monsterName;}
+    public string Temperament { get => temperament; }
+    public string Description { get => description;}
+    public Sprite ImageOriginal { get => imageOriginal; }
+    public Sprite ImageCleared { get => imageCleared; }
+    public bool Completed { get => completed;}
+
+    [SerializeField] private string monsterName;
+    [SerializeField] [TextAreaAttribute] private string temperament;
+    [SerializeField] [TextAreaAttribute] private string description;
+    [SerializeField] private Sprite imageOriginal;
+    [SerializeField] private Sprite imageCleared;
+    [SerializeField] private bool completed;
+
+    private CompendiumManager m_compendiumManager;
+
```

Comment:
Just to keep things consistent with the layout of other scripts, the public properties will need to move to the bottom of the declaration section.

```suggestion
    [SerializeField] private string monsterName;
    [SerializeField] [TextAreaAttribute] private string temperament;
    [SerializeField] [TextAreaAttribute] private string description;
    [SerializeField] private Sprite imageOriginal;
    [SerializeField] private Sprite imageCleared;
    [SerializeField] private bool completed;

    private CompendiumManager m_compendiumManager;

    public string MonsterName { get => monsterName;}
    public string Temperament { get => temperament; }
    public string Description { get => description;}
    public Sprite ImageOriginal { get => imageOriginal; }
    public Sprite ImageCleared { get => imageCleared; }
    public bool Completed { get => completed;}
```